app/models/sir_trevor_rails/block.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+require 'ostruct'
+
+module SirTrevorRails
+  # Forked from (former) upstream sir-trevor-rails 0.6.2 gem:
+  # https://github.com/madebymany/sir-trevor-rails/blob/931b9554f5268158b4da8817477cdc82e4e2e69c/lib/sir_trevor_rails/block.rb
+  # Copyright (c) 2013-2014 by ITV plc - http://www.itv.com
+  class Block < OpenStruct
+    DEFAULT_FORMAT = :markdown
```

Comment:
Doable to not use `OpenStruct` here? or not worth the effort currently

Reply:
Nope. Each type of block can have its own data.